hazelcast/proxy/map.py

```diff
@@ -212,6 +501,12 @@ def put(self, key, value, ttl=-1):
         return self._put_internal(key_data, value_data, ttl)
 
     def put_all(self, map):
+        """
+        Copies all of the mappings from the specified map to this map. No atomicity guarantees are
+        given. It could be that in case of failure some of the key/value-pairs get written, while others are not.
+
```

Comment:
...given. In case of a failure, some of the key-value pairs may get written, while others are not.